web/ui/react-app/src/TimeInput.tsx

```diff
@@ -32,19 +32,24 @@ interface TimeInputProps {
 
 class TimeInput extends Component<TimeInputProps> {
   private timeInputRef = React.createRef<HTMLInputElement>();
-  private $time: any | null = null;
+  private $time: any = null;
 
   getBaseTime = (): number => {
     return this.props.time || moment().valueOf();
   };
 
+  calcTimeRange = () => {
+    const { range } = this.props;
+    return range * 1000 + (range > 10 ? 10000 : 0);
```

Comment:
I don't see how this behaves the way we want. E.g. if I have a range of 60s, my range is now shifted by 70s - what is the purpose of that?

I think we still want the old behavior (at least for the graph) where we shift by 1/2 of the configured range, so that enough (half) of the old portion of a graph is still visible when shifting the data to the left or right.

Reply:
As I mention here https://github.com/prometheus/prometheus/pull/6460#issuecomment-565755035
I misunderstood the way it's working in the old UI so I'll revert all this and just will add the seconds timestamp

Reply:
Ok cool.